Feature/danger

ardiadrianadri opened the pull request:
Pull request to add a Continuous Integration process

ardiadrianadri committed:
Remove yarn from circle configuration

ardiadrianadri committed:
Add new version of nodejs

ardiadrianadri committed:
Add danger task

ardiadrianadri committed:
Fix bug circle configuration

ardiadrianadri committed:
Fix error in the danger command

ardiadrianadri committed:
Fix danger command

ardiadrianadri committed:
Add public token

adriadrianardi commented:
Check that I can publish comments

ardiadrianadri committed:
Update danger token

ardiadrianadri committed:
Other token

ardiadrianadri committed:
Add more circle configuration

ardiadrianadri committed:
Add new circle configuration

ardiadrianadri committed:
Otra vez lo mismo por enesima vez

ardiadrianadri committed:
Otra prueba mas

ardiadrianadri committed:
Add my personal token

ardiadrianadri committed:
Mas tokens

ardiadrianadri committed:
More configuration

adriadrianardi commented:
<!--
  1 failure:  The pull request ...
  0 warning: 
  1 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The pull request should have a description</td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Changed Files in this PR: 
 - .circleci/config.yml- dangerfile.js- package.json</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d8f76d966e4f642bcee58e48396981ffb579367e
</p>

ardiadrianadri committed:
Add check if the PR has a person assigned

ardiadrianadri committed:
The PR must have a description

ardiadrianadri committed:
Add branch name validation

ardiadrianadri committed:
Add more test in the danger file

- Add test to check that the issue has, at least, one reviewer
- Add test to check that the PR is related to a task

ardiadrianadri committed:
Add Contribution document

uRuiz commented:
Formación de Danger para el 18/05/2019
Se cancela la pull request
